src/Polysemy/ConstraintAbsorber.hs

```diff
@@ -0,0 +1,62 @@
+{-# LANGUAGE AllowAmbiguousTypes         #-}
+{-# LANGUAGE ConstraintKinds             #-}
+{-# LANGUAGE FlexibleInstances           #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving  #-}
+{-# LANGUAGE MultiParamTypeClasses       #-}
+{-# LANGUAGE ScopedTypeVariables         #-}
+{-# LANGUAGE TypeFamilies                #-}
+{-# LANGUAGE UndecidableInstances        #-}
+{-# LANGUAGE UndecidableSuperClasses     #-}
+{-# OPTIONS_GHC -fplugin=Polysemy.Plugin #-}
```

Comment:
This is already specified in the cabal file, so unnecessary here.

Reply:
Done.  Sorry, I usually check for unnec. pragmas and forgot.  Do we need the plugin bit in each file?

Reply:
Done.  Sorry, I usually check for unnec. pragmas and forgot.  Do we need the plugin bit in each file?

Reply:
Done.  Sorry, I usually check for unnec. pragmas and forgot.  Do we need the plugin bit in each file?

Reply:
The plugin is enabled globally across all of the zoo, so no need for it anywhere.